src/app/space/create/deployments/resource-usage/deployments-resource-usage.component.html

```diff
@@ -11,11 +11,25 @@ <h3>
             </b>
           </h3>
         </div>
-        <div [collapse]="resourcesCollapsed">
-          <div class="col-xs-12 col-sm-4 col-md-3" *ngFor="let environment of environments | async">
-            <resource-card [spaceId]="spaceId | async" [environment]="environment"></resource-card>
+        <div *ngIf="(environments.first() | async); else loading">
+          <div [collapse]="resourcesCollapsed">
+            <div class="col-xs-12 col-sm-4 col-md-3" *ngFor="let environment of environments | async">
+              <resource-card [spaceId]="spaceId | async" [environment]="environment"></resource-card>
+            </div>
           </div>
         </div>
+        <ng-template #loading>
+          <div>
+            <div [collapse]="resourcesCollapsed">
+              <div class="col-xs-12 col-sm-4 col-md-3">
+                  <loading-resource-card></loading-resource-card>
+                </div>
+                <div class="col-xs-12 col-sm-4 col-md-3">
+                  <loading-resource-card></loading-resource-card>
+                </div>
+            </div>
+          </div>
+        </ng-template>
```

Comment:
I think this part can be cleaned up and streamlined a bit. The `*ngIf` being around the `[collapse]` seems funny since the collapsible section is there whether or not placeholders are being displayed. The columns containing the cards are also the same size whether they are real cards or placeholders.

Also, since you are actually using the result of the environments observable in this template (as opposed to the template in the other commit), there's no need to use `.first()` on it here. You can use the `environments | async as environmentsList; else loading` syntax for example, and then change the `*ngFor` to iterate over `environmentsList` rather than `environments | async`.